src/components/Dashboard/Modals/EditAppModal/Tabs/IntegrationTab/Components/InputElements/IntegrationSelector.tsx

```diff
@@ -87,9 +87,14 @@ export const IntegrationSelector = ({ form }: IntegrationSelectorProps) => {
     },
     {
       value: 'pihole',
-      image: 'https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons@master/png/pihole.png',
+      image: 'https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons@master/png/pi-hole.png',
       label: 'PiHole',
     },
+    {
+      value: 'adGuardHome',
+      image: 'https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons/png/adguard-home.png',
```

Comment:
Is there a reason to use master for one and nothing for the other link?

Reply:
Pretty sure, that it will just default to master (the default branch). I'll adjust the rest of the links